README.md

```diff
@@ -6,39 +6,54 @@
     <p align="center">Lint your feature flag config files.</p>
 </p>
 
-This action will perform the linting of the gofeatureflag config file.
+The `go-feature-flag/gofeatureflag-lint-action` Github action performs the
+linting of the [gofeatureflag](https://gofeatureflag.org/) flag config file. It
+checks for errors in the config file and returns the error message if any.
 
 📢 Join Our Community: [![Slack](https://img.shields.io/badge/join-us%20on%20slack-gray.svg?longCache=true&logo=slack&colorB=green)](https://gophers.slack.com/messages/go-feature-flag)
 
-## Inputs
-
-`config-root`: The root folder for the config flag. The default value for the
-folder is `./config`.
-## Outputs
-
-`lint-message`: The value will be blank if no error is found. If an error is
-found, the value will be the error message.
-## Example usage
+## Usage 📝
 
 ```yaml
 on: [push]
 
 jobs:
-  feature-flag-lint:
+  gofeatureflag-lint-job:
     runs-on: ubuntu-latest
-    name: A job to say hello
+    name: A demo job to run gofeatureflag-lint on a config file
     steps:
-      # To use this repository's private action,
-      # you must check out the repository
+      
+      ## checkout the code from the repository
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Hello world action step
-        uses: go-feature-flag/gofeatureflag-lint-action@__hash_value__
-        id: hello          
+      
+      ## run the gofeatureflag-lint action
+      - name: Lint the config file
+        uses: go-feature-flag/gofeatureflag-lint-action@ea0524d562003105471e2ccdf3485e98db23fa48
+
+        ## set the input parameters for the action
         with:
-          config-root: ${GITHUB_WORKSPACE}
+          flagFilePath: ${{ github.workspace }}/config/flag-config.yaml
+          flagFiletype: 'yaml'
```

Comment:
All examples of Github Action I see around are using [kebab case](https://www.theserverside.com/definition/Kebab-case) for parameters.

I will change this into something like this:
```suggestion
          flag-file: ${{ github.workspace }}/config/flag-config.yaml
          format: yaml
```